configure

```diff
@@ -20884,6 +20887,23 @@ $as_echo "#define SDL_VIDEO_DRIVER_RPI 1" >>confdefs.h
     fi
 }
 
+CheckRockchip()
```

Comment:
I don't think this needs to be in it's own function. Removing the CheckRockchip() wrapped function will guarantee that if a user passes --enable-video-rockchip that it's picked up.

Reply:
If you want to use this, I believe you'll need to add the call to it on line 25806

Reply:
Fixed.
You buildroot toolchain can pass --enable-video-rockchip during the build process?

Reply:
Yes, buildroot's sdl2 package passes the values dynamically based on the config for the compile. It doesn't have a rockchip option yet, but using kmsdrm as an example it would add "--enable-video-kmsdrm" to the configure options when the config includes BR2_PACKAGE_SDL2_KMSDRM. I've added the setting for rockchip to test when all the changes are in and ready.

Reply:
Can you give it a try with buildroot fetching my repo and passing ./configure --enable-video-kmsdrm --enable-video-rockchip --disable-video-x11 --disable-video-vulkan  --disable-video-wayland ?

In X11 SDL is now using 3D HW accel as before.

Unfortunately, SDL_HINT_VIDEO_X11_FORCE_EGL = 1 as suggested by @cgutman does not work as SDL switch to SW Rendering.
![Screenshot_2022-01-04_18-10-39](https://user-images.githubusercontent.com/16623039/148125139-1e44eb11-daae-4895-99fe-6df349e0db1b.png)

Reply:
Still waiting for my other comment to be addressed before doing a compile :) Pretty confident you need to set enable_video_rockchip=yes in the else condition.

Reply:
I can't set enable_video_rockchip=yes until there is a way to **#undef** a previously set variable with AC_DEFINE.
**If someone knows a way to do it in configure.ac** (and configure) then we can set rockchip with default to yes and not break GLX for others.